### ISSUE TITLE
error run create_data.sh , only generate the Soft connection，the size is 57 bytes , but did not genereate the actual lmdb  

### ISSUE THREAD

**Issue by 19931991** [MhLiao/TextBoxes#81]
error run create_data.sh , only generate the Soft connection，the size is 57 bytes , but did not genereate the actual lmdb  .
 
this is the ｒｅｓｕｌｔ of running the create_data.sh : 
![image](https://user-images.githubusercontent.com/28551961/34662842-82c71124-f48e-11e7-9482-428e10c796f0.png)
and my data is : 
![image](https://user-images.githubusercontent.com/28551961/34662867-9af0cd6c-f48e-11e7-8d36-39623fcb66ec.png)
the crate_data.sh is : 
![image](https://user-images.githubusercontent.com/28551961/34662884-b019dd28-f48e-11e7-94cf-2bea7713613b.png)

please tell me how to solve the problem ! It troubled me so long ! Thank you  very much !

**Comment by 19931991** [comment]
@MhLiao

**Comment by MhLiao** [comment]
I suggest you check your trainval.txt and test.txt

**Comment by 19931991** [comment]
Thanks for your reply , I try to generate the trainval.txt and test.txt ｆollowing the online tutoria . l think my trainval.txt and test.txt are right .And this is my trainval.txt : 
![image](https://user-images.githubusercontent.com/28551961/34751227-4ad48612-f5e6-11e7-83df-a6b556ccd103.png)

![image](https://user-images.githubusercontent.com/28551961/34751247-5f03ef06-f5e6-11e7-9dd6-4652932ee114.png)

![image](https://user-images.githubusercontent.com/28551961/34751256-6db0aff8-f5e6-11e7-8f52-b10a1d5f4313.png)

![image](https://user-images.githubusercontent.com/28551961/34751261-7932291a-f5e6-11e7-95ad-42b9b7a2b105.png)

Looking forward to your reply! @MhLiao

**Comment by MhLiao** [comment]
It seems that there is no error in your scripts. How about the source of the soft connection? The true data should be in "$data_root_dir/$dataset_name/$db/"

**Comment by 19931991** [comment]
Thanks for your reply , my lmdb is 
![image](https://user-images.githubusercontent.com/28551961/34754179-2a3534d6-f5f7-11e7-974f-c32eafe15f87.png)
 
and my soft connetection is 
![image](https://user-images.githubusercontent.com/28551961/34754202-446e9400-f5f7-11e7-946e-06deb89f6e22.png)

I do not know how to debug , please help me ! @MhLiao

**Comment by MhLiao** [comment]
@19931991 I can not find the mistakes. Can you create Pascal VOC data successfully?

**Comment by 19931991** [comment]
Yes, I create Pascal VOC data successfully , besides,  I download the voc2007 and voc2012 , and when I run create_data.sh to create lmdb data ， it still did not succeed , the result is the same as my description - only empty soft connection.  Can you tell me What environment is needed in create the lmdb ? Besides , I try to use other's lmdb (Target detection rather than text recognition) to train the network . the network runs up normally. In other words, if my data of lmdb is maked successfully , my work is done . So there is some wrong with my environment and cause this result ? Thank you very much !

**Comment by 19931991** [comment]
@MhLiao

**Comment by 19931991** [comment]
Can I create lmdb in other computer , and copy to my computer to use it ? @MhLiao

**Comment by MhLiao** [comment]
Yes, you can.

**Comment by qiuxiaqing** [comment]
I have recently encountered the same problem. Have you solved it?  If you solve this problem, can you share the method? Thank you

**Comment by tecsai** [comment]
You may see [https://github.com/BVLC/caffe/issues/6159#issuecomment-709894376](url)
